### ISSUE TITLE
Is Lootr compatible with Lost Cities's chests?

### ISSUE THREAD

**Issue by NiGHT0FDAWN** [McJty/LostCities#554]
The chests in TLC aren't being converted in my minecraft world...

Is Lootr compatible with Lost Cities's chests?
If not, is there any way to make it compaitble that I can do?

**Comment by strubium** [comment]
No, Lootr is not. The Lost Cities Hardcodes its loot tables so I could see it overriding Lootr. You could try making an addon with [this tutorial](https://www.youtube.com/watch?v=Ck9_yknPx4c&ab_channel=JorritTyberghein).

**Comment by McJty** [comment]
I wouldn't know why lootr isn't compatible with Lost Cities because it uses standard loot tables but what you can do is to configure the chests in Lost Cities yourselves. .Lost Cities uses a datapack system for the loot tables

**Comment by Patard0** [comment]
> I wouldn't know why lootr isn't compatible with Lost Cities because it uses standard loot tables but what you can do is to configure the chests in Lost Cities yourselves. .Lost Cities uses a datapack system for the loot tables

How can i do that?? i don´t really know anything about minecraft modding but at least changing .cfg files is easy, i've been searching for something at the config files about the loot and i couldn't see anything, am i missing something or simply i have to modify another stuff? (im using 1.16.5 forge 36.2.41)

**Comment by McJty** [comment]
All documentation here:

https://www.mcjty.eu/docs/mods/lost-cities/
